bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx93-es.html

```diff
@@ -0,0 +1,57 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros-protocol.html" import picto with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+{% block body_class %}wnp93-v3{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_93') }}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox">
+  {% include 'firefox/whatsnew/includes/header.html' %}
+
+  {% set landing_source = '?source=' + campaign if campaign else '' %}
+
+  <section class="wnp-content-main">
+    <div class="mzp-l-content mzp-t-content-lg">
+      <h1 class="wnp-pre-title">Es el mes de la ciberseguridad</h1>
+      <h2 class="wnp-main-title">Es el momento de disfrutar de auténtica privacidad online</h2>
+
+      <div class="wnp-main-image video-container">
+        <video id="video" preload="metadata" autoplay loop poster="{{ static('img/firefox/whatsnew/whatsnew93/hero-v3.svg') }}">
+          <source src="{{ static('img/firefox/whatsnew/whatsnew93/hero-v3.webm') }}" type="video/webm">
```

Comment:
Is it worth putting this on assets.mozilla.net?

Reply:
I thought about that, but since these are pretty small videos pmac thinks they should be fine serving from the regular CDN. If they were really big files I wouldn't want to add them to the repo, but they're all under 300k (webm video is smaller and seems better than an equivalent gif).